testing-with-jest.test.js

```diff
@@ -1,27 +1,40 @@
-// ASSESSMENT 1: Testing practical questions
-
+// April Rieger's // ASSESSMENT 1: Testing practical questions
+//
 // $ yarn add jest
-
+//
 // Practice red-green refactor development
 // $ yarn jest
 
 
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
-
-
-
-
+// describe a function that takes an argument that adds two numbers
+describe("addThemUp", () => {
+  //test that the outcome is as expected
+  test('adds 2+2 to equal 4', () => {
+    //expect the outcome
+    expect(addThemUp.add(2, 2)).toBe(4);
+  })
+})
 // --------------------1b) See the test fail. THEN write the code to make the test pass.
 
-
-
-
-
-// --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
-
-
-
-
-
+const addThemUp = {
+  add: (num1, num2) => num1 + num2
+}
+console.log(2, 2)
```

Comment:
the syntax on this funciton is a little funky, and the function call isn't completely done yet. you might have meant something like this:

```javascript
const addThemUp = (num1, num2) => {
  return num1+num2
}

console.log(addThemUp(2, 2))
```